templates/taxi/manufacturer_create.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+<h1>Manufacturer {{ object|yesno:"update,create" }}</h1>
```

Comment:
The `yesno` filter is used with `object` to determine whether to display 'update' or 'create'. However, `object` might not be defined in the context for a create view. Consider using a condition to check if the object exists or use a different context variable to determine the action.